include/dpp/webhook.h

```diff
@@ -56,6 +56,24 @@ class DPP_EXPORT webhook : public managed {
 	 */
 	webhook();
 
+	/**
+     * @brief Construct a new webhook object using the Webhook URL provided by Discord
+    */
```

Comment:
need to document the url parameter:
`@param webhook_url a fully qualified web address of an existing webhook`